lib/Controller/AssistantApiController.php

```diff
@@ -108,9 +126,14 @@ public function getAssistantTask(int $metaTaskId): DataResponse {
 	/**
 	 * Get user's tasks
 	 *
-	 * @param string|null $taskType
-	 * @param int|null $category
+	 * Get a list of assistant tasks for the current user.
+	 *
+	 * @param string|null $taskType Task type id. If null, tasks of all task types will be retrieved (default: null)
+	 * @param int|null $category Task category. If null, tasks of all categories will be retrieved (default: null)
```

Comment:
```suggestion
	 * @param string|null $taskType Task type id. If null, tasks of all task types will be retrieved
	 * @param int|null $category Task category. If null, tasks of all categories will be retrieved
```
The default value will already be in the spec and displayed correctly